HOW/social/process/_requirements.md

```diff
@@ -0,0 +1,34 @@
+---
+Status: <DRAFT|REVIEW|REJECTED|SUPERSEDED|DEFERRED|LIVE>
```

Comment:
Is there also `ABANDONED` or something like that, for a proposal that for any reason is no longer relevant, but nothing superseded it?

Reply:
Yep, good point.

Reply:
What do you think of merging ABANDONED with REJECTED as per below?

Reply:
Yep, that makes sense to me. I'm good with either name for that merged concept.